src/schemas/Entitlement/FeaturePrivilegeObject.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+required:
+  - code
+properties:
+  code:
+    type: string
+    example: "max"
+    description: "Unique code for the privilege."
+  name:
+    type:
+      - string
+      - "null"
+    example: "Maximum"
+    description: "Display name for the privilege."
+  value_type:
+    type: string
+    enum:
+      - integer
+      - boolean
+      - string
+      - select
+    example: "integer"
+    description: "Data type of the privilege value. Default: string"
+  config:
```

Comment:
I think there should be a oneOf here where the config with select_options is present only if value_type is select.